dt-backend/src/model/user/user.service.ts

```diff
@@ -1,9 +1,21 @@
-import { Injectable } from '@nestjs/common';
+import {Injectable} from '@nestjs/common';
 import { prisma } from '../../main';
 import { User } from '../types/user.type';
+import { Prisma } from '@prisma/client'
 
 @Injectable()
 export class UserService {
+
+  async User(userWhereUniqueInput: Prisma.UserWhereUniqueInput): Promise<User> {
```

Comment:
Let's try to come up with better (non-autogenerated) names ^^!
It's a bit painful to read, tbh - especially if our goal is to create clean, maintainable code!

Reply:
updated the name on this